common/app/layout/Front.scala

```diff
@@ -170,7 +173,8 @@ object FaciaContainer {
     config: CollectionConfigWithId,
     collectionEssentials: CollectionEssentials,
     containerLayout: Option[ContainerLayout],
-    componentId: Option[String]
+    componentId: Option[String],
+    omitMPU: Boolean = false
```

Comment:
Yeah, I've had too many run ins with defaults to use them anymore, especially in case classes.
Don't mind them too much on methods!